subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -1158,8 +1159,8 @@ void ull_conn_tx_demux(u8_t count)
 			break;
 		}
 
-		conn = ll_conn_get(lll_tx->handle);
-		if (conn->lll.handle == lll_tx->handle) {
+		conn = ll_connected_get(lll_tx->handle);
+		if (conn) {
```

Comment:
No need to check `lll.handle` vs `lll_tx->handle`?

Reply:
it is checked in ll_connected_get, which is doing the same as old code.